src/app/components/publications/publication-algorithms-list/publication-algorithms-list.component.ts

```diff
@@ -0,0 +1,114 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { EntityModelPublicationDto } from 'api/models/entity-model-publication-dto';
+import { AlgorithmService } from 'api/services/algorithm.service';
+import { PublicationService } from 'api/services/publication.service';
+import { GenericDataService } from '../../../util/generic-data.service';
+
+@Component({
+  selector: 'app-publication-algorithms-list',
+  templateUrl: './publication-algorithms-list.component.html',
+  styleUrls: ['./publication-algorithms-list.component.scss'],
+})
+export class PublicationAlgorithmsListComponent implements OnInit {
+  @Input() publication: EntityModelPublicationDto;
+  algorithms: any[] = [];
```

Comment:
Try to avoid any if possible, the generated code should contain the proper types